react/features/video-quality/components/VideoQualityDialog.web.js

```diff
@@ -115,13 +115,21 @@ class VideoQualityDialog extends Component {
     render() {
         const { _audioOnly, _p2p, t } = this.props;
         const activeSliderOption = this._mapCurrentQualityToSliderValue();
+        const showP2PWarning = _p2p && !_audioOnly;
 
         return (
             <div className = 'video-quality-dialog'>
                 <h3 className = 'video-quality-dialog-title'>
                     { t('videoStatus.callQuality') }
                 </h3>
-                { !_audioOnly && _p2p ? this._renderP2PMessage() : null }
+                <div className = { showP2PWarning ? '' : 'hide-warning' }>
+                    <InlineMessage
```

Comment:
Just curious. Why did you decide to get rid of the _renderP2PMessage() method? It looks like you can still add the div and keep the message rendering in a separate method? I don't mind, either way, I was just wondering.